support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -115,9 +115,7 @@ export function CheckoutNudgeContainer({
 		defaultAmount,
 	).toString();
 
-	const isDynamic = useContributionsSelector(
-		isUserInAbVariant('makeItAnnualNudgeGlobal', 'variant'),
-	);
+	const isDynamic = Country.isVatAffected(countryId);
```

Comment:
Shouldn't isDynamic be `true` if they're not in a VAT affected country? 

`const isDynamic != Country.isVatAffected(countryId)`...?